src/Compilers/Server/VBCSCompiler/VBCSCompiler.cs

```diff
@@ -13,7 +13,7 @@ internal static class VBCSCompiler
     {
         public static int Main(string[] args)
         {
-            using var logger = new CompilerServerLogger();
+            using var logger = new CompilerServerLogger("VCBCSCompiler");
```

Comment:
typo

Reply:
Almost missed it too. Easy to slip by

---
In reply to: [613327259](https://github.com/dotnet/roslyn/pull/52572#discussion_r613327259) [](ancestors = 613327259)